test/e2e/e2e.go

```diff
@@ -157,6 +157,21 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 
 		FileSystemId = id
 		ginkgo.By(fmt.Sprintf("Created EFS filesystem %q in region %q for cluster %q", FileSystemId, Region, ClusterName))
+		//Sleep 5 minutes to allow file system bootstrapping
+		ginkgo.By("Sleeping 5 minutes to allow file system bootstrapping")
+		time.Sleep(5 * time.Minute)
+		ginkgo.By("Done sleeping. Now proceeding to test access point creation")
+		accessPointId, err := c.CreateAccessPoint(FileSystemId, ClusterName)
+		if err != nil {
+			framework.ExpectNoError(err, "creating access point")
```

Comment:
not sure if this check will work. from the e2e logs it seems like creation silently succeeds. But it's worth a try.

Reply:
Looks like creating is not enough, I'll need to mount file system using access points too for all the boot strapping to complete.